src/de/grobox/liberario/fragments/DirectionsFragment.java

```diff
@@ -533,7 +564,7 @@ private void showMore(boolean animate) {
 
 	private void showLess(boolean animate) {
 		showingMore = false;
-		ui.via.setLocation(null);
+		ui.via.setLocation((Location)null);
```

Comment:
I haven't found a real solution for this. I just moved the need for the cast into a method of the LocationView. I don't know if this makes it better.